src/Sylius/Component/Core/Uploader/PathGeneratorInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Component\Core\Uploader;
+
+use Gaufrette\Filesystem;
```

Comment:
For now I am fine with being coupled to Gaufrette, I don't want to put too much effort into this part of Sylius because we have #496 + a lot of things to consider.